src/sentry/static/sentry/app/components/assigneeSelector.tsx

```diff
@@ -25,18 +24,23 @@ import GroupStore from 'app/stores/groupStore';
 import MemberListStore from 'app/stores/memberListStore';
 import ProjectsStore from 'app/stores/projectsStore';
 import space from 'app/styles/space';
-import {
-  Actor,
-  SuggestedAssignee,
-  SuggestedOwner,
-  SuggestedOwnerReason,
-  Team,
-  User,
-} from 'app/types';
+import {Actor, SuggestedOwner, SuggestedOwnerReason, Team, User} from 'app/types';
 import {buildTeamId, buildUserId, valueIsEqual} from 'app/utils';
 
+type SuggestedAssignee = Actor & {
+  suggestedReason: SuggestedOwnerReason;
+  assignee: AssignableTeam | User;
+};
+
+type AssignableTeam = {
+  id: string;
+  display: string;
+  email: string;
+  team: Team;
+};
+
 type Props = {
-  id: string | null;
+  id: string;
```

Comment:
are you sure about removing the `null` type? I noticed that the defaultProps still set it to `null`

https://github.com/getsentry/sentry/blob/a0738f5575126a4c05bdf0f54819a39b11912509/src/sentry/static/sentry/app/components/assigneeSelector.tsx#L67

maybe we could make it optional and remove it from the defaultProps object?

Reply:
Removed the default prop, everywhere that uses it seems to be typed as string, and all the places we use it don't allow for it to be null.